app/helpers/toolbar_helper.rb

```diff
@@ -0,0 +1,28 @@
+module ToolbarHelper
+  def toolbar(title:, subtitle: '', &block)
+    content_tag :div, class: 'toolbar-container' do
+      content_tag :div, id: 'toolbar' do
+        dom_title(title, subtitle) + dom_toolbar do
+          yield if block_given?
+        end
+      end
+    end
+  end
+
+  protected
+
+  def dom_title(title, subtitle)
+    content_tag :div, class: 'title-container' do
+      heading = content_tag(:h2, title.html_safe, title: strip_links(title))
+      next heading if subtitle.blank?
+      heading + content_tag(:p, subtitle, class: 'subtitle')
+    end
+  end
+
+  def dom_toolbar(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `dom_toolbar(*)` if you want the method to accept any arguments but don't care about them.